vic/vic_run/src/func_surf_energy_bal.c

```diff
@@ -750,7 +750,7 @@ func_surf_energy_bal(double  Ts,
     }
     Evap = 0.;
     if (!SNOWING) {
-        if (VEG && veg_var->fcanopy > 0) {
+        if (VEG) { // if VEG is true, then fcanopy > 0 and LAI > 0
```

Comment:
I would put comment above in a separate line for consistency

Reply:
Done.